src/applications/vaos/sass/vaos.scss

```diff
@@ -47,3 +47,9 @@ h1:focus {
     top: 40%;
   }
 }
+
+.vaos__form-button {
+  @media (max-width: $xsmall-screen - 1) {
```

Comment:
Generally we prefer to write these mobile-first, so setting it to width: 100% by default, then removing that style above 320px.